statsig.js

```diff
@@ -28,6 +28,56 @@ export default (function () {
         return;
       }
 
+      // measure cdn perf
+      setTimeout(async () => {
+        const start = performance.now();
+        await fetch(
+          'https://latest.api.statsig.com/v1/download_config_specs?k=client-LAx5juseYG9kxnB2vHLxFluaFmZVv9aAKPmw1NB8rps', {cache: "reload"}
+        )
+          .then(() => {
+            const delta = performance.now() - start;
+            window.statsig.logEvent('cloud_cdn_latency', delta);
+          })
+          .catch((e) => {
+            const delta = performance.now() - start;
+            window.statsig.logEvent('cloud_cdn_latency', delta, {
+              error: e.message,
+            });
+          });
+      }, 1000);
+      setTimeout(async () => {
+        const start = performance.now();
+        await fetch(
+          'https://cache.statsigcdn.com/v1/download_config_specs?k=client-LAx5juseYG9kxnB2vHLxFluaFmZVv9aAKPmw1NB8rps', {cache: "reload"}
+        )
+          .then(() => {
+            const delta = performance.now() - start;
+            window.statsig.logEvent('cloudflare_cdn_latency', delta);
+          })
+          .catch((e) => {
+            const delta = performance.now() - start;
+            window.statsig.logEvent('cloudflare_cdn_latency', delta, {
+              error: e.message,
+            });
+          });
+      }, 1000);
+      setTimeout(async () => {
+        const start = performance.now();
+        await fetch(
+          'https://dcs-worker.statsig.workers.dev/v1/download_config_specs/client-LAx5juseYG9kxnB2vHLxFluaFmZVv9aAKPmw1NB8rps.js', {cache: "reload"}
+        )
+          .then(() => {
+            const delta = performance.now() - start;
+            window.statsig.logEvent('cloudflare_worker_latency', delta);
+          })
+          .catch((e) => {
+            const delta = performance.now() - start;
+            window.statsig.logEvent('cloudflare_worker_latency', delta, {
+              error: e.message,
+            });
+          });
+      }, 1000);
```

Comment:
maybe stagger these so they dont all start at the same time?

Reply:
done